src/css/map-view.css

```diff
@@ -475,11 +479,85 @@ represents 1 unit of the given distance measurement. */
   line-height: 100%;
 }
 
+/*****************************************************************************************
+ *
+ * Search Input
+ *
+ * Search Input is a shared component for searching information in the map toolbar.
+ *
+ */
+
+
+.search-input {
```

Comment:
@yvonnesjy I was going to remind you that we don't use any CSS preprocessors, but then saw that the rules were being applied and had to look this up! There is native CSS nesting now??? what??? 🎉🎉🎉 Thank you for showing me this!

Reply:
ooooooh that's why nothing was nested! I guess all projects I worked on use preprocessors? cause I didn't know this was ever not an option haha. google says nesting was supported natively since last year🙂

Reply:
ooooooh that's why nothing was nested! I guess all projects I worked on use preprocessors? cause I didn't know this was ever not an option haha. google says nesting was supported natively since last year🙂

Reply:
Amazing! Yes, before that, this was a syntax that was used in SCSS or Sass, and you would need to compile it to CSS for browser use. Perhaps MetacatUI will get away with never adopting a CSS preprocessor...